core/ledger/kvledger/txmgmt/privacyenabledstate/db_test.go

```diff
@@ -150,7 +151,7 @@ func TestGetStateMultipleKeys(t *testing.T) {
 func testGetStateMultipleKeys(t *testing.T, env TestEnv) {
 	env.Init(t)
 	db := env.GetDBHandle(generateLedgerID(t))
-
+	defer env.Cleanup()
```

Comment:
nit - `defer` should immediately follow `env.Init()` (especially when you are doing this at other places already)

Reply:
completed.